common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -443,14 +445,20 @@ class @Problem
     if not @submit_save_waitfor(@save_internal)
       @disableAllButtonsWhileRunning @save_internal, false
 
+  focus_on_save_notification: =>
+    if @saveNotification.length > 0
+      @saveNotification.focus()
+
   save_internal: =>
     Logger.log 'problem_save', @answers
     $.postWithPrefix "#{@url}/problem_save", @answers, (response) =>
       saveMessage = response.msg
       if response.success
         @el.trigger('contentChanged', [@id, response.html])
-      @gentle_alert saveMessage
-      @updateProgress response
+        @render(response.html, @focus_on_save_notification)
+        @updateProgress response
```

Comment:
This was preexisting, it should probably be moved out of this and placed back where it was. Should I change the existing logic?